README.md

```diff
@@ -1,3 +1,13 @@
-# Contributing
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+# omi-kits
+
+This repository contains binary distributions for the
+[omi](https://github.com/Microsoft/omi) repository. Only the latest
+version of OMI should be checked into this repository at any one time.
+
+This repository exists so older UNIX systems can easily get binary kits
+of the OMI binaries in an automated way. Note that older UNIX systems do
+not necessarily contain the `curl` or `wget` tools.
```

Comment:
No longer splitting into chunks, compress did the job for now. Will follow up in person.